src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,10 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        String cleanText = text.replaceAll("[^a-zA-Z0-9]", "");
```

Comment:
```suggestion
        String cleanText = text.replaceAll("[^a-zA-Z0-9]", "").toUpperCase();
```

Reply:
I call cleanText.toUpperCase() in return statement. Fixed, but is it a mistake?

Reply:
at the previous version, you call it twice, but now you can do it one time

Reply:
got it, thank you)
fixed